openshift-install-powervs

```diff
@@ -1432,6 +1456,20 @@ function main {
       warn "Enabling forceful destruction option for terraform destroy command"
       FORCE_DESTROY=1
       ;;
+    "-flavor")
+      shift
+      flavor="$1"
+      if [[ ! -s "$varfile" ]]; then
+	for((var = 1; var <= $#; var++)); do
+          if [ ${!var} == "var.tfvars" ]; then
+	    varfile="${!var}"
+	    echo $varfile
+	  fi
+	done
+      fi
+      validate_flavor "$flavor"
+      vars+=" -var-file ../$varfile -var-file ../automation/compute-vars/$flavor.tfvars"
```

Comment:
All this should go in `precheck_input` function. Here just validate the flavor argument.

Reply:
Please remove this line from here.